mlflow/store/tracking/file_store.py

```diff
@@ -162,7 +162,7 @@ def __init__(self, root_directory=None, artifact_root_uri=None):
         """
         Create a new FileStore with the given root directory and a given default artifact root URI.
         """
-        super().__init__()
+        super(FileStore, self).__init__()
```

Comment:
to test autofix

Reply:
Good. Failed as expected.

<img width="756" alt="image" src="https://github.com/mlflow/mlflow/assets/17039389/654e2fab-72ac-46d8-8414-31021d0db555">